SampleApps/CurrencyRecogniser/CurrencyRecogniser.Android/MainActivity.cs

```diff
@@ -8,12 +8,18 @@
 using Android.OS;
 using Plugin.CurrentActivity;
 using Xam.Plugins.OnDeviceCustomVision;
+using System.IO;
 
 namespace CurrencyRecogniser.Droid
 {
     [Activity(Label = "CurrencyRecogniser", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
     public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
     {
+        bool useFilesForTheModel = false;
```

Comment:
@jimbobbennett 
I added this flag to switch between assets and files for the model